crates/p2p_protocol/src/p2p.rs

```diff
@@ -0,0 +1,163 @@
+use crate::{error::P2pResult, p2p_capnp};
+
+/// a message used to verify connectivity with a remote node
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, Hash)]
+pub struct MsgPing {
+    /// set the milliseconds since unix epoch when sending this message
+    pub ping_send_epoch_ms: u64,
```

Comment:
`ping_` part of the variable name seems unnecessary

Reply:
:+1: